app/controllers/users_controller.rb

```diff
@@ -13,28 +13,28 @@ def create
     @user = User.new(user_params)
```

Comment:
on the user listing page
`def index
    @users = User.all
  end`
how are these users sorted?
Sort them with id desc

Reply:
Done